src/components/ArticleComplexCard.jsx

```diff
@@ -57,13 +64,25 @@ export default function ArticleCard({article, users}) {
       setComments(await articleComments.comments)
   }
 
-  const renderOutStuff = ()=>{
+  const handleArticleVote = ()=>{
+    let increment 
+    if (hasVoted)
+      increment = -1  
+    else 
+      increment = 1
+    setArticleVotes((currentVote)=> currentVote+increment)
+    setHasVoted((voted)=> !voted)
+    
+    axios.patch(`https://skelbon-news-api.onrender.com/api/articles/${article.article_id}`, { inc_votes : increment }).catch((err)=>{setIsVoteError('Network error - unable to update vote - try later')})
```

Comment:
Consider having all your api calls stored in another file so they can be used across the app